### ISSUE TITLE
Defining a second entry generates an error

### ISSUE THREAD

**Issue by snake3lake** [deric/puppet-pubkey#2]
**Code**:
```
pubkey::ssh { johndoe_ed25519':
    type    => 'ed25519',
    comment => 'johndoe_ed25519',
    tags    => ['tag_users'],
}

pubkey::ssh { 'johndoe_ed25519-sk':
    type    => 'ed25519-sk',
    comment => 'johndoe_ed25519-sk@foobar',
    tags    => ['tag_users'],
  }
```

**Error**:
```
Error: Could not retrieve catalog from remote server: 
Error 500 on SERVER: Server Error: Evaluation Error: 
Error while evaluating a Resource Statement, Evaluation Error: Error while evaluating a Resource Statement, Duplicate declaration: File[/var/cache/pubkey] is already declared at (file: /etc/puppetlabs/code/environments/production/modules/pubkey/manifests/ssh.pp, line: 87); 
cannot redeclare (file: /etc/puppetlabs/code/environments/production/modules/pubkey/manifests/ssh.pp, line: 87)
 (file: /etc/puppetlabs/code/environments/production/modules/pubkey/manifests/ssh.pp, line: 87, column: 5) 
(file: /etc/puppetlabs/code/environments/production/modules/my/manifests/ssh.pp, line: 12)
```

**Comment by deric** [comment]
Should be fixed in `v0.8.0`.

**Comment by snake3lake** [comment]
Yes, it is. Thanks!
